openfl/utilities/optimizers/torch/fedprox.py

```
@@ -67,14 +75,22 @@ def __init__(
         if weight_decay < 0.0:
             raise ValueError(f"Invalid weight_decay value: {weight_decay}")
         if mu < 0.0:
```

One liner `assert mu >= 0.0, f"FedProx regularizer coefficient must be greater than or equal to 0, got {mu}"` is sufficient. 
I don't think there are any scenarios where negative `mu` is used. It is OK to raise exception here instead of a warning.